shared/is-type.sh

```diff
@@ -2,11 +2,18 @@
 
 set -e
 ARGV_TYPE="$1"
+ARGV_DIRECTORY="$2"
 set -u
 
 [[ "${DEBUG}" == "false" ]] || set -x
 
-pushd source
+pushd $ARGV_DIRECTORY
+
+if [ -f repo.yml ]; then
+  REPO_TYPE=$(yq read repo.yml 'type')
+fi
+
+[[ $REPO_TYPE == $ARGV_TYPE ]] && exit 0
```

Comment:
Careful here. If `repo.yml` does not contain a `type` and `$1` == "" then this returns `true` and exists with 0

Reply:
As discussed via Flowdock, this case won't happen since the `yq` version currently installed in the CI is outdated and returns the `null` string when a key is not present. We have to be careful once we update `yq`